assets/js/book.js

```diff
@@ -0,0 +1,32 @@
+function Book(title, author, pages, read) {
+  this.id = localStorage.uniqueId;
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+
+  this.info = function() {
```

Comment:
This isn't the best way to add methods into `Book`. Doing it this way will cause the application to re-declare these methods everytime we need them.

What we should do is add this methods into `Book.prototype`.